helpers/or.js

```diff
@@ -9,21 +9,12 @@ const _ = require('lodash');
  * {{#or 1 0 0 0 0 0}} ... {{/or}}
  */
 const factory = () => {
-    return function() {
-        var args = [],
-            opts,
-            any;
-
-        // Translate arguments to array safely
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
-        }
-
+    return function(...args) {
         // Take the last argument (content) out of testing array
```

Comment:
This comment seems incorrect now

Reply:
I thought it made sense as arguments (non-keyword) are passed to functions.

Reply:
Maybe if it said "options" instead of "content", it would make more sense.

Reply:
fixed

Reply:
Nvm rebasing issues . . .

Reply:
Git is behaving weird but I believe things are fixed now.

Reply:
whitespace in test case of for.js is fixed in the other pr.

Reply:
fixed here now too actually.

Reply:
Actually realized that I missed a file.

Reply:
Three files have been changed to be more explicit about what is being pulled out of the args array.